_layouts/release.html.haml

```diff
@@ -67,7 +67,7 @@
           %a.artifact{:href=>a.to_javadoc_url, :target=>'_blank'} javadoc
         %a.artifact{:href=>a.to_pom_url, :target=>'_blank'} pom
 - unless page.release_notes.nil?
-  %h3
+  %h3.release-notes
```

Comment:
I'm not UI developer nor HTML guru, so I can be wrong, but for adding additional information about the element I would use attributes.